src/secrets/CommandEdit.ts

```diff
@@ -54,41 +55,85 @@ class CommandEdit extends CommandPolykey {
           },
           logger: this.logger.getChild(PolykeyClient.name),
         });
-        const response = await binUtils.retryAuthentication(
-          (auth) =>
-            pkClient.rpcClient.methods.vaultsSecretsGet({
-              metadata: auth,
-              nameOrId: secretPath[0],
-              secretName: secretPath[1],
-            }),
-          meta,
-        );
-        const secretContent = response.secretContent;
+
         const tmpDir = await fs.promises.mkdtemp(
           path.join(os.tmpdir(), 'polykey-'),
         );
         const tmpFile = path.join(tmpDir, 'pksecret');
```

Comment:
I feel like the name of the secret we're editing rather than a hard coded `pksecret` name. This is because editors will tell you the name of the file it's editing and it should match up for convenience sake. Its not a big issue though.